htdocs/core/tpl/card_presend.tpl.php

```diff
@@ -207,9 +207,8 @@
 			$formmail->withtoccuser = $listeuser;
 		}
 	}
-
 	$formmail->withto = $liste;
-	$formmail->withtofree = (GETPOSTISSET('sendto') ? (GETPOST('sendto') ? GETPOST('sendto') : '1') : '1');
+	$formmail->withtofree = (GETPOSTISSET('sendto') ? (GETPOST('sendto', 'none') ? GETPOST('sendto', 'none') : '1') : '1');
```

Comment:
the GETPOST with no parmaete is using 'alpha' by default. It seems enough.
Why using 'none' ?

Reply:
In the case of an email presend, the email addresses of the recipients are in the "<aaaa@aaaa.fr>" format.
The default 'alpha' parameter of the GETPOST deletes the chevrons as well as the contents inside the chevrons.

Reply:
With 'alpha', the code is 
$out = str_replace(array('"', '../'), '', trim($out));
So it remove only ../ and double quotes. But < and > remains. Can you check.

Reply:
In v12, it seems the < and > stay with 'alpha'. So the PR looks useless. Are you sure.
In v13, it is true but PR introduce security hole, so i introduced a new "alphawithlgt" key for GETPOST and pushed a PR in v13 for the mentionned line.

Reply:
Indeed, the chevrons stay with 'alpha'.
I should have filled in the parameter 'alpha' instead of 'none' in my PR.
The GETPOST with no parameter is using 'alphanohtml' by default, which erases the chevrons.